src/views/product-integrations-landing/index.tsx

```diff
@@ -27,6 +28,18 @@ export default function ProductIntegrationsLanding({
 	breadcrumbLinks,
 	product,
 }: ViewProps) {
+	/**
+	 * Show a loading spinner until 1500ms after the `useEffect` runs to prevent a
+	 * hydration error.
+	 */
+	const [isLoadingClient, setIsLoadingClient] = useState(true)
+	useEffect(() => {
+		setTimeout(() => {
+			// TODO uncommonent before merging
+			// setIsLoadingClient(false)
+		}, 1500)
```

Comment:
Is there anyway we can link this to the actual api call to know when we have the data to render? I assume the dynamic call is what's causing the flash?

Reply:
@kendallstrautman the API call is happening server-side and the data is being passed into the view component. The flash and hydration mismatch errors are happening when there are query parameters in the page URL. The query parameters affect what HTML is rendered on the page, and since those aren't accessible until the client starts loading, there's a brief flash from the default/unfiltered view of this page to the filtered view.